src/components/availability-panel/drag-drop-context/DraggableComponent.tsx

```diff
@@ -33,6 +33,14 @@ const DraggableComponent: FC<draggableProps> = ({
   title = '',
 }) => {
   const draggableIds = useContext(disableDrag);
+  const [draggableImageUrl, setDraggableImageUrl] = useState(DUMMY_PROFILE);
+  const draggableImageOnError = useCallback(() => setDraggableImageUrl(DUMMY_PROFILE), []);
+
+  useEffect(() => {
+    getAssigneeProfilePic(draggableId).then((res) => {
```

Comment:
why is `.catch` missing?

also in case of `res` comes empty how are we handling that?

Reply:
+1